app/survey_config/business_config.py

```diff
@@ -14,7 +14,8 @@ class BusinessSurveyConfig(
     SurveyConfig,
 ):
     base_url: str = BASE_URL
-    account_service_url: str = f"{BASE_URL}/surveys/todo"
+    account_service_url: str = f"{BASE_URL}/sign-in/logout"
```

Comment:
Any reason this was changed(I might have missed it on the card)? /surveys/todo is the portfolio page for RAS/RM